jcvi/apps/base.py

```diff
@@ -37,6 +37,20 @@
 os.environ["LC_ALL"] = "C"
 JCVIHELP = "JCVI utility libraries {} [{}]\n".format(__version__, __copyright__)
 
+_logging_debug = logging.debug
+def debug_with_location(message: str):
```

Comment:
```suggestion
def debug_with_location(message: str, *args: Any):
```

Reply:
Test is failing due to many places invoking with multiple args, e.g. `logging.debug("message: %s", message)`